libs/sdk-ui-charts/src/highcharts/chart/HighChartsRenderer.tsx

```diff
@@ -202,15 +200,15 @@ export default class HighChartsRenderer extends React.PureComponent<
     }
 
     // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    public onChartSelection(event: any) {
+    onChartSelection = (event: any): void => {
```

Comment:
Please keep `public` and `private` for these functions

Reply:
ok! fixed.